### PR TITLE
Prep for Q1 DevTools survey.

### PR DESCRIPTION
**Pull request by kenzieschmoll**
@cobblest @jacob314

### CODE REVIEW

**Comment by kenzieschmoll** [comment]
Before landing this, some changes to the devtools_server may be required.

**Comment by jacob314** [comment]
Any update on landing this?

File: packages/devtools_app/lib/html_main.dart [hunk 1]
```diff
@@ -65,11 +65,12 @@ void main() {
...
-      // November 14, 2019 (4 weeks after the survey start date
-      // October 17, 2019).
-      final surveyThresholdDate = DateTime(2019, 11, 14);
-      if (DateTime.now().isBefore(surveyThresholdDate)) {
+      // Show the Q1 2020 DevTools survey if the current date falls in range.
+      final surveyStartDate = DateTime(2020, 3, 11);
+      final surveyEndDate = DateTime(2020, 4, 10);
+      final now = DateTime.now();
+      if (now.isAfter(surveyStartDate) && now.isBefore(surveyEndDate)) {
+        await ga.setActiveSurvey('Q1-2020');
```
Comment (terrylucas) [inline packages/devtools_app/lib/html_main.dart]: setActiveSurvey can return false if any error creating .devtiols first time, persisting the properties or filesystem problem.  Nothing specific just success or failure.

Comment (kenzieschmoll) [reply packages/devtools_app/lib/html_main.dart]: added a check for this. thanks

**Comment by kenzieschmoll** [comment]
Fixes https://github.com/flutter/devtools/issues/1605

**Pull request merged**